cli/foundation-models/system/inference/phi-3-mini-v/phi-3-mini-v-128k-instruct/online-endpoint-deployment.sh

```diff
@@ -0,0 +1,79 @@
+set -x
+
+# script inputs
+subscription_id="<SUBSCRIPTION_ID>"
+resource_group_name="<RESOURCE_GROUP>"
+workspace_name="<WORKSPACE_NAME>"
+
+# This is the model from system registry that needs to be deployed
+registry_name="azureml"
+model_name="Phi-3-mini-v-128k-instruct-patched-vllm"
```

Comment:
Use the right model name

Reply:
Shouldn't the mode name be "Phi-3-vision-128k-instruct"
https://ml.azure.com/registries/azureml-preview/models/Phi-3-vision-128k-instruct/version/1